src/multi_robot_multi_goal_planning/planners/rrtstar_base.py

```diff
@@ -1216,10 +1216,10 @@ def InformedInitialization(self, mode: Mode) -> None:
             self.informed[mode].initialize()
 
     def SampleNodeManifold(self, mode:Mode) -> Configuration:
-        if np.random.uniform(0, 1) <= self.p_goal:
+        if np.random.uniform(0, 1) > self.p_goal:
```

Comment:
While I agree that all of these things are now as expected (I think), the time I had to spend thinking about it suggests that some parts might be better rewritten a bit:

```
if rnd < p_goal:
   goal_sampling
else:
   same stuff as now
```